examples/parse.rs

```diff
@@ -0,0 +1,13 @@
+//! Let's you easily try out the tokenizer with e.g.
+//! printf '<h1>Hello world!</h1>' | cargo run --example=parse
+use html5gum::Tokenizer;
+use std::io::Read;
+
+fn main() {
+    let mut html = String::new();
+    std::io::stdin().read_to_string(&mut html).unwrap();
+
+    for token in Tokenizer::new(&html).infallible() {
```

Comment:
in 0.2 it should be possible to read from any BufRead directly

Reply:
Ah, very neat!